CHANGES.md

```diff
@@ -14,6 +14,9 @@ We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (
 * Bump default `ktlint` version to latest `1.2.1` -> `1.3.0`. ([#2165](https://github.com/diffplug/spotless/pull/2165))
 * Bump default `ktfmt` version to latest `0.49` -> `0.51`. ([#2172](https://github.com/diffplug/spotless/pull/2172))
 * Rename property `ktfmt` option `removeUnusedImport` -> `removeUnusedImports` to match `ktfmt`. ([#2172](https://github.com/diffplug/spotless/pull/2172))
+* Bump default `eclipse` version to latest `4.29` -> `4.32`. ([#2179](https://github.com/diffplug/spotless/pull/2179))
+* Bump default `greclipse` version to latest `4.49` -> `4.31`. ([#2179](https://github.com/diffplug/spotless/pull/2179))
```

Comment:
We cannot update to 4.32 here yet, because, currently, the Groovy formatter version is tied to the Eclipse version and the matching plugin version has not been released yet (https://github.com/groovy/groovy-eclipse/milestone/25).